### ISSUE TITLE
Cannot signup because email is already used by an existing user

### ISSUE THREAD

**Issue by Nomane** [InfoSec812/sonar-auth-google#23]
Hi,

I have configured the plugin in this way :

sonar.auth.googleoauth.allowUsersToSignUp=true
sonar.auth.googleoauth.clientId.secured=XXXXXXX.apps.googleusercontent.com
sonar.auth.googleoauth.clientSecret.secured=XXXXXXX
sonar.auth.googleoauth.enabled=true
sonar.auth.googleoauth.loginStrategy=Unique
sonar.auth.googleoauth.limitOauthDomain=mycompany.com

That's works fine on the first connection, but when I retry to login again I have this error message:

![image](https://user-images.githubusercontent.com/7596282/61065313-b761ae00-a403-11e9-8c72-adea2b022ad9.png)

Any advice ?

Thanks

**Comment by InfoSec812** [comment]
No, I have not seen any error like that in the past. I'll try to experiment and see if I can reproduce it, but I cannot give you an estimate of how soon that might happen.
